src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioIntegrationTestUtilities.csproj

```diff
@@ -104,6 +105,10 @@
       <Project>{e2e889a5-2489-4546-9194-47c63e49eaeb}</Project>
       <Name>Diagnostics</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\Test\Utilities\Portable\TestUtilities.csproj">
```

Comment:
@tannergooding is this a dependency we want to take? I know we've discussed making this be a standalone library. @dpoeschl what did you need it for?